lib/tasks/exercise_test_task.rb

```diff
@@ -47,6 +47,6 @@ def run_exercise_tests_in(dir)
   def test_options
     return '' unless ARGV.include?('--')
 
-    ARGV.drop_while { |e| e != '--' }.drop(1).join(' ')
+    Rake.arguments.join(' ')
```

Comment:
Wow, yeah, I missed that... relying on the tests I thought were there.  And the manual run at that time working.

Thanks @Insti